.jenkins/pytorch/multigpu-test.sh

```diff
@@ -22,5 +22,5 @@ time python test/run_test.py --verbose -i distributed/test_jit_c10d
 time python test/run_test.py --verbose -i distributed/test_distributed_fork
 time python test/run_test.py --verbose -i distributed/test_c10d
 time python test/run_test.py --verbose -i distributed/test_c10d_spawn
-time python test/run_test.py --verbose -i distributed/rpc/test_tensorpipe_agent
+time python test/run_test.py --verbose -i distributed/rpc/cuda/test_tensorpipe_agent
```

Comment:
I just realized that we are not running `process_group` tests on multi-gpu CI. Is this intentional?

Reply:
Not sure, the TensorPipe tests were added in https://github.com/pytorch/pytorch/pull/49210 by @pritamdamania87. I think that TensorPipe has some "native" multi-GPU tests, whereas for ProcessGroup (which is CPU-only) it would just be the DDP part that is multi-GPU. Perhaps this was the reason not to add them here?

Reply:
Yep, it could be. And since we are retiring ProcessGroup soon, it should be fine to leave it out.